### DISCUSSION TITLE
[outline.nvim] No response from provider when requesting symbols !

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#4175] [Q&A]
This error is displayed when I try to trigger Outline when the lsp is not fully loaded. 

I can understand this behavior, but when I trigger the Outline again after the lsp is loaded, it still returns an error. 

And when I open other files of the same type, I still get an error. 

This does not occur when the Outline is triggered only after the vim is restarted and the lsp is completely loaded.

In my memory, there was a time when this was not the case. 

When I trigger the Outline before the lsp is loaded, it does not report an error immediately. Instead, it waits until the lsp is loaded and then the outline window pops up.

I don't know why it can't be like this later. 

I'm not sure whether it's because of LazyVim or outline.nvim.

**Comment by iamxiaojianzheng** [comment]
![image](https://github.com/user-attachments/assets/68a1cb34-db7b-45c5-b098-fee6dd6521d9)

![image](https://github.com/user-attachments/assets/17e486c6-d18e-42b7-9bcb-39f66ab41327)

**Comment by folke** [comment]
Thats not an issue with LazyVim
